staff/juan-sensio/cv/index.html

```diff
@@ -1,42 +1,68 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="es">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" href="styles.css">
-    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
     <title>CV</title>
 </head>
 <body>
-    <h1>Aitor Tilla</h1>
+    <header>
+        <nav id="top">
+            <a href="#datos">Datos Personales</a>
+            <a href="#educacion">Educación</a>
+            <a href="#experiencia">Experiencia</a>
+            <a href="#habilidades">Habilidades</a>
+        </nav>
+    </header>
 
-    <div>
-        <h2>Datos Personales</h2>
-        <p>Carrer de Roc Boronat, 35 (Barcelona)</p>
-        <p>698203948</p>
-        <p>aitortilla@gmail.com</p>
-    </div>
-
-    <div>
-        <h2>Educación</h2>
-        <h3>Skylab coders (www.skylabcoders.com). </h3>
-        <p>Full-Stack Web Development Bootcamp.</p>
-        <h3>La Universidad de la Calle (UDC).</h3>
-        <p>No tengo la E.S.O, pero mi abuela dice que soy muy espabilado.</p>
-    </div>
-    
-    <div>
-        <h2>Experiencia</h2>
-        <h3>NASA</h3>
-        <p>Desarrollo del software de aviónica del programa Space Shuttle.</p>
-    </div>
+    <main>
+        <h1>Aitor Tilla</h1>
+        
+        <section id="datos">
+            <h2>Datos Personales</h2>
+            <p>Carrer de Roc Boronat, 35 (Barcelona)</p>
+            <p>698203948</p>
+            <p><mark>aitortilla@gmail.com</mark></p>
+        </section>
     
-    <div>
-        <h2>Habilidades</h2>
-        <p>Git nivel avanzado (pull request legend).</p>
-        <p>Beber chupitos con el codo.</p>
-    </div>
+        <section id="educacion">
+            <h2>Educación</h2>
+            <section>
+                <h3>Skylab coders (www.skylabcoders.com). </h3>
+                <p><mark>Full-Stack</mark> Web Development Bootcamp.</p>
+            </section>
+            <aside>2018</aside>
```

Comment:
why not ```<time>``` instead of ```<aside>``` and located inside its own ```<section>```

Reply:
@manuelbarzi change done.